lib/hiptest-publisher/utils.rb

```diff
@@ -51,7 +51,7 @@ def make_url(options)
 def fetch_project_export(options)
   url = make_url(options)
 
-  open(url, "User-Agent" => 'Ruby/hiptest-publisher', :ssl_verify_mode => OpenSSL::SSL::VERIFY_NONE)
+  open(url, "User-Agent" => 'Ruby/hiptest-publisher', :ssl_verify_mode => OpenSSL::SSL::VERIFY_PEER)
```

Comment:
verify_peer was all that was needed?  Didn't have to call `OpenSSL::X509::Store#set_default_paths` like in that SO post?